packages/rum-core/src/domain/view/trackViews.spec.ts

```diff
@@ -936,35 +935,33 @@ describe('view event count', () => {
   })
 
   describe('update view name', () => {
-    it('should update an undefined view name if the experimental feature is enabled', () => {
-      mockExperimentalFeatures([ExperimentalFeature.UPDATE_VIEW_NAME])
+    it('should update an undefined view name', () => {
       viewTest = setupViewTest({ lifeCycle })
 
-      const { getViewUpdate, startView, updateViewName } = viewTest
+      const { getViewUpdate, startView, setViewName } = viewTest
 
       startView()
-      updateViewName('foo')
+      setViewName('foo')
       expect(getViewUpdate(3).name).toEqual('foo')
     })
 
     it('should update a defined view name if the experimental feature is enabled', () => {
```

Comment:
**suggestion:**
we could use a more appropriate description since this is not an experimental feature any more. And use "set view name" phrasing instead of "update view name"